code/_onclick/hud/radial.dm

```diff
@@ -28,6 +28,8 @@ GLOBAL_LIST_EMPTY(radial_menus)
 		closeToolTip(usr)
 
 /obj/screen/radial/slice/Click(location, control, params)
+	if(!parent) //Orphaned somehow
+		return
 	if(usr.client == parent.current_user)
 		if(next_page)
 			parent.next_page()
```

Comment:
Why not just use ?. here
Also how would this get orphaned?

Reply:
Not sure where ?. would go. Which line?

No idea how, but it did: `Runtime in radial.dm, line 31: Cannot read null.current_user`

Reply:
I'd reccomend reading up on ?. but yes it basically does this

Reply:
The operation is self evident, if the var to the left of the `?.` is null return null otherwise continue on to the member field, but since you felt compelled to point it out I'll go ahead and just assume you want it changed and do that.

IMO a separate early return is easier to read.